packages/expo-updates/android/src/androidTest/java/expo/modules/updates/UpdatesUtilsInstrumentationTest.java

```diff
@@ -27,4 +31,34 @@ public void testParseDateString_writtenTimezoneWithColon() throws ParseException
     Assert.assertEquals(new Date(1605053874699L), UpdatesUtils.parseDateString("2020-11-11T00:17:54.699+00:00"));
     Assert.assertEquals(new Date(1605050274699L), UpdatesUtils.parseDateString("2020-11-11T00:17:54.699+01:00"));
   }
+
+  @Test
+  public void testGetMapFromStringifiedJSON_empty() throws Exception {
+    Map<String,String> emptyMap = getMapFromStringifiedJSON("{}");
+    Assert.assertEquals(emptyMap, new HashMap<>());
+  }
+
+  @Test
+  public void testGetMapFromStringifiedJSON_expectedFormat() throws Exception {
+    Map<String, String> expected = new HashMap<>();
+    expected.put("expo-channel-name","main");
+
+    Map<String,String> emptyMap = getMapFromStringifiedJSON("{'expo-channel-name':'main'}");
```

Comment:
JSON strings need to be double-quoted... I'm surprised this works at all.
```suggestion
    Map<String,String> emptyMap = getMapFromStringifiedJSON("{\"expo-channel-name\":\"main\"}");
```